liquidctl/driver/aquacomputer.py

```diff
@@ -93,7 +93,7 @@ def initialize(self, **kwargs):
         """
 
         fw = self.firmware_version
-        serial_number = self.serial_number
+        serial_number = self._serial_number
```

Comment:
Shouldn't this be `self._serial`?  And how isn't this tripping the tests?

---

Also, needing two reads in the normal path (which is to call `initialize` first) isn't ideal, even if the interval between reports is fairly short.

I think it would be better if the properties used a shared `_read_device_statics` (or something like it) method to parses both fields from a single `_read`.

Reply:
`self._serial` is where the serial number gets stored from within the `self._serial_number` property (currently), it would be empty on its own.

Reply:
Oh, yeah, of course!